lib/private/Files/Stream/Encryption.php

```diff
@@ -315,6 +315,23 @@ public function stream_read($count) {
 		return $result;
 
 	}
+	
+	/**
+	* stream_read wrapper to read complete requested block
+	*/
+	private function stream_read_block($blockSize) {
```

Comment:
```suggestion
	private function stream_read_block(int $blockSize) {
```

Should be save to typehint $blockSize